### ISSUE TITLE
Sending confirmation of application to candidates

### ISSUE THREAD

**Issue by olasitarska** [DjangoGirls/djangogirls#25]
> This issue is a part of [Project: application process](https://github.com/DjangoGirls/djangogirls/wiki/Project:-Application-process) & the work is happening on [applications branch](https://github.com/DjangoGirls/djangogirls/tree/applications)

:warning: This depends on #21 and #22.

That would be a very useful feature -- when someone applies, we send them a confirmation with all their answers on email. Sometimes people wrote to us in the past asking if we received their application, becuase they don't have confirmation and are not sure.

**Comment by olasitarska** [comment]
Plain text emails again!

**Comment by robcharlwood** [comment]
Sorting this now as this is a quick win.

**Comment by olasitarska** [comment]
:+1:

**Comment by robcharlwood** [comment]
@olasitarska When I looked, this was already done in the ``applications`` branch but was a bit broken. I have fixed the broken-ness and raised a pull request. Once that gets merged this ticket is done. :+1:

**Comment by parodia** [comment]
@olasitarska we came to conclusion that organizers should have the possibility to edit the confirmation mail as many events have website (so I bet also mails) in their national languages so currently working on it

**Comment by robcharlwood** [comment]
@olasitarska @parodia We just need to find a neat way to interpolate the dynamic data without confusing the organisers on the front end with ``{{foo.bar.car.dar}}`` style mark up.

**Comment by olasitarska** [comment]
hm, organizers can edit questions in Question model, so we can just get the questions from there, right?

**Comment by robcharlwood** [comment]
@olasitarska this is not for the answers themselves. :) 

This is for the standard thank you text e.g

Hi there!

This is a confirmation of your application to <a href="http://djangogirls.org/{{ application.form.page.url }}">{{ application.form.page.title }}</a>. Yay! That's a huge step already, we're proud of you!

Mind that this is not a confirmation of participation in the event, but a confirmation that we received your application.

You'll receive an email from the team that organizes {{ application.form.page.title }} soon. You can always reach them by answering to this email or by writing to {{ application.form.emails_send_from }}.

For your reference, we're attaching your answers below.

Hugs, cupcakes and high-fives!
Django Girls

**Comment by olasitarska** [comment]
Make sense!
